src/metric/definition.go

```diff
@@ -839,6 +840,7 @@ var KSMQueries = []prometheus.Query{
 	},
 	{MetricName: "kube_hpa_status_current_replicas"},
 	{MetricName: "kube_hpa_status_desired_replicas"},
+	{MetricName: "kube_node_info"},
```

Comment:
Should we have some tests for that?

Reply:
Good point, I'm not sure where those would fit with the current architecture, since I haven't seen any obvious place for them. Do you have any suggestion?

Reply:
Given the simplicity of the code and complexity of adding tests for this, I'm fine leaving it out for now :smile: 

The question was more of a bait rather than a requirement :smile: